java-kubectl/README.md

```diff
@@ -0,0 +1,100 @@
+# Payroll
+
+This guides shows an example of developing a Spring Boot Java application with cnd. It's based on [Spring's gs-rest-service example](https://github.com/spring-guides/gs-rest-service).
+
+## Deploy the payroll service
+
+Clone the repo and move to this example folder.
+
+```console
+git clone https://github.com/okteto/examples
+cd examples/java-kubectl
+```
+
+Deploy the Payroll application by using the following command:
+```console
+kubectl apply -f manifests
+```
+
+Wait for one or two minutes until the application is running. Run the following command to get the External IP of the service:
+```console
+kubectl get service payroll
+```
+
+## Cloud native development
+
+In order to activate your Cloud Native Development, execute:
+
+```console
+cd payroll
+cnd up
+```
+
+This will create a remote container which is synchronized with your local code changes. The container already includes java dev tools (e.g. gradle), and it's configured to  automatically compile your code directly on the cluster everytime you make a change.
```

Comment:
I wouldn't say create, I would say it replaces the prod container (reusing the same k8 manifest) or something like that.